assets/javascripts/src/components/Navigation.jsx

```diff
@@ -21,17 +21,42 @@ export default class Navigation extends React.Component {
         const isFirstPage = !processing && this.props.page === PAGES.CONTRIBUTION;
         const showMobileBack = !processing && this.props.page == PAGES.PAYMENT;
         const showBack = !processing && this.props.page !== PAGES.CONTRIBUTION;
-        const showPay = !processing && !!this.props.amount && this.props.page === PAGES.PAYMENT;
-        const showForward = !processing && this.props.page == PAGES.DETAILS;
+        const showPay = !processing && !!this.props.amount && this.props.page === (this.props.stripeCheckout?PAGES.DETAILS:PAGES.PAYMENT);
+        const showNext = (!this.props.stripeCheckout) && !processing && this.props.page == PAGES.DETAILS;
+        const classes = {
```

Comment:
not sure defining the classes in the object and then referencing them in the tag makes things any clearer